LICENSE

```diff
@@ -0,0 +1,427 @@
+Attribution-ShareAlike 4.0 International
+
+=======================================================================
```

Comment:
A problem:

There's non-CC content here. Like the social icons:

* https://github.com/neuropoly/neuro.polymtl.ca/blob/52903f3245aa23228dc3d6ba7b637db3dbd16fc5/.gitbook/assets/github-mark-20px.png
* https://github.com/neuropoly/neuro.polymtl.ca/blob/52903f3245aa23228dc3d6ba7b637db3dbd16fc5/.gitbook/assets/mail-142.png
* https://github.com/neuropoly/neuro.polymtl.ca/blob/52903f3245aa23228dc3d6ba7b637db3dbd16fc5/.gitbook/assets/logo_youtube.png

I derived this template from https://github.com/executablebooks/sphinx-book-theme which is BSD 3-Clause

* https://github.com/neuropoly/neuro.polymtl.ca/blob/master/_templates/topbar/repobuttons.html

So I think we actually need to add multiple sections to this license file, mentioning which files are covered by which license?

Reply:
I think so, but we can try to keep it as easy as possible by limiting the others license to files we already have and even replace them if it's possible to reduce special cases.

Reply:
I agree, replacing content if we have to would help a lot.